### ISSUE TITLE
NTR: Babesia PCR positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8086]
New term request Babesia PCR positive (nadea3_181012160942): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000736
- label: Positive bloodstream babesia nucleic acid pathogen test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of genus Babesia (parasite) in the blood circulation by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8086
- synonyms:
